clog/clog.go

```diff
@@ -193,6 +193,48 @@ func infof(ctx context.Context, lastErr bool, publicLog bool, format string, arg
 	}
 }
 
+// Info logs a message with key-value pairs in a slog-like style.
+// Example: Info(ctx, "hello", "key1", value1, "key2", value2)
+// This will log: "hello key1=value1 key2=value2"
+func Info(ctx context.Context, msg string, keyvals ...interface{}) {
```

Comment:
I wonder if we don't make our `clog.go` more confusing with this change. Normally, you'd think that `Info()` is the same as `Infof()` but without the formatting values inside. However, this implementation makes `Info()` behaves totally different than `Infof()`. 

@mjh1 Wdyt? You recently did some work re:logging.

Reply:
Not sure that I understand the concern. The behavior matches that of `slog.Info` . The `f` convention to indicate a format string is used elsewhere in golang, eg log.Info / Infof, fmt.Errorf, etc. We are not doing anything different here. Personally, I like the brevity of plain `Info` and consistency / ease of use compared to munging format strings.

Reply:
Yeah I kind of see the concern it's two different syntaxes when usually libraries seem to support one or the other, but for our case I think it's ok. We want to be able to use the newer style syntax, but also want to be able to use our existing context implementation. The other thing is that this method can still be used as you described, just for a simple log line without values, just don't pass any keyvals.